### DISCUSSION TITLE
tasmota32 networking setup

### DISCUSSION THREAD

**Discussion by msetina** [arendst/Tasmota#20043] [General]
After setting up WIFI and static IP tasmota32 13.2.0 somehow still has default ip 192.168.4.1 and looks like 2 IPs enabled.

heat-storage-K-5012 (192.168.44.43, 192.168.4.1)

In tasmota AP is turned off after WIFI setup, but not in tasmota32. I stumbled onto this when debuging MQTT not connecting with rc -2, but no error on mosquittto...

**Comment by msetina** [comment]
after saving on Config Wifi web page something in network setup changed and MQTT connects. Networking was set with command over serial. I am afraid I will have to change that and use autoconfig. It is not reliable.

**Reply by msetina** [reply]
Wificonfig is on value 4

**Reply by sfromis** [reply]
The AP mode default IP being part of the Tasmota header does not mean that it is enabled, there is no "default IP" for STA mode. In other words, don't worry. You can see active IP addresses on the "Information" page (below the header), and the `Status 5` command.

**Reply by msetina** [reply]
I mentioned it (being confused) because normal tasmota disables the 192.168.4.1 IP

humidity-sensor3-7672 (192.168.44.42)

**Reply by sfromis** [reply]
Technically, Tasmota does not "disable" 192.168.4.1 in any cases, as there simply is no "default IP" for STA mode, **only** for AP mode. The "oddity" is that the AP mode IP is available in the different Wifi library for ESP32, even outside of AP mode. Tasmota reports what's there, without this meaning enabled/active.
It would become active in case of switching to AP mode, where I then can see `(0.0.0.0, 192.168.4.1)`, no STA mode IP address.

**Reply by msetina** [reply]
Is there a way to disable 192.168.4.1?

**Reply by msetina** [reply]
OK. So on tasmota32 this is how it stays... ON the other hand after my setup procedure networking was in a bad state as MQTT did not connect. I realy feel I need to move to autoconfig file for ESP32 boards. Now I do not know how to make it clean.

**Reply by sfromis** [reply]
As 192.168.4.1 is not enabled in STA mode, there is nothing to disable. And you cannot tweak the UI to not include this in the header, unless you use `SetOption53` to completely exclude host name and IP address from the heder. 

Whatever happens to your MQTT connection issue is unrelated to the AP mode IP address being visible, and while using an autoconfig setup is generally a good strategy, I'd not expect this to make a difference to MQTT connection issues.

**Reply by msetina** [reply]
Understood. Thank you. I think that MQTT issue was a network setup issue. Trying to find what the problem is I found the dengling IP. The 192.168.4.1 was just a confusion.
I need to clean up the setup as I do not know WTH happened to networking. It is probably connected to my last discussion #19821
